src/core/cli/initializer.ts

```diff
@@ -154,7 +155,10 @@ class Initializer {
         'What language would you like to use in browser?',
         conf.browserLanguage
       );
-
+      conf.chromeOptions = await this.promptFolders(
+        'What chrome options would you like to use in browser?',
```

Comment:
More matching: "Which chrome options would you like to use in the browser?"
or with Tomek's suggestions:  "Which additional chrome options arguments would you like to use in a browser?"